notebooks/ai-ml/HuggingFaceEmbeddingGenerationInSpark.ipynb

```diff
@@ -0,0 +1,772 @@
+{
```

Comment:
<h4>Create new Interactive session</h4><p>Once the template is created, select the interactive template as the kernel for the notebook. This will create a Dataproc Interactive Session, which you can check the status of&nbsp;in the <a href="https://console.cloud.google.com/dataproc/interactive?" target="_blank">console</a>. This will take a few minutes to finish.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudDataproc/cloud-dataproc/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>